### DISCUSSION TITLE
How to use rigtools to turn off extension

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1176] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
What do I run to turn off extension?

**Comment by stuartcat** [comment]
I don't think you can disable them if you dont have management on the extension if you do it should show switches to turn it off.

**Reply by Balistic123** [reply]
What do you mean by management?

**Comment by akabutnicer** [comment, marked as answer]
Working with Writable on the project, I found a way to get this to work. When pressing Q to evaluate on an extension, enter the ID: `gndmhdcefbhlchkhipcnnbkcmicncehk`. In doing so, you should see all of your extensions available to be enabled and disabled at will.
